regimes/es/scenarios.go

```diff
@@ -46,6 +48,28 @@ var invoiceTags = common.InvoiceTagsWith([]*tax.KeyDefinition{
 			i18n.ES: "Contribuyente en régimen simplificado",
 		},
 	},
+	{
+		Key: TagFacturaE,
+		Name: i18n.String{
+			i18n.EN: "FacturaE",
+			i18n.ES: "FacturaE",
+		},
+		Desc: i18n.String{
+			i18n.EN: "Invoice must comply with the FacturaE standard.",
+			i18n.ES: "La factura debe cumplir con el estándar FacturaE.",
+		},
+	},
+	{
+		Key: TagTicketBAI,
```

Comment:
The conclusion here for TicketBAI is to move this complexity over to the conversion tool so that the user makes a specific choice about which region they are declaring in.

Reply:
Right on! Working on this next

Reply:
Done here:
* https://github.com/invopop/gobl.ticketbai/pull/11
* https://github.com/invopop/ticketbai/pull/17

Reply:
Done here:
* https://github.com/invopop/gobl.ticketbai/pull/11
* https://github.com/invopop/ticketbai/pull/17